frost-core/src/round2.rs

```diff
@@ -90,9 +90,23 @@ where
         verifying_share: &frost::keys::VerifyingShare<C>,
         lambda_i: Scalar<C>,
         challenge: &Challenge<C>,
+        group_commitment: &frost::GroupCommitment<C>,
+        verifying_key: &frost::VerifyingKey<C>,
     ) -> Result<(), Error<C>> {
+        let mut commitment_share = group_commitment_share.0;
+        let mut vsh = verifying_share.0;
+        if <C>::is_need_tweaking() {
+            commitment_share = <C>::tweaked_group_commitment_share(
+                &group_commitment_share.0,
+                &group_commitment.0
+            );
+            vsh = <C>::tweaked_verifying_share(
+                &verifying_share.0,
+                &verifying_key.element
```

Comment:
these "tweaks" have a different meaning as the taproot tweaks right? I'm confused :D

Reply:
Hi! I'm sorry for a pause for reply.

Yes, it's not taproot, maybe need more correct names to `fn`.